csrc/host_ir/executor.cpp

```diff
@@ -46,10 +46,8 @@ void HostIrExecutor::handle(PostOnStream* post_ir) {
   Expr* op = post_ir->hostOpToPost();
   if (op->isA<HostUnit>()) {
     postCompute(post_ir);
-  } else if (op->isA<Communication>()) {
-    postCommunication(post_ir);
   } else {
-    NVF_ERROR(false, "The op cannot be posted on a stream: ", op);
+    dispatch(op);
```

Comment:
This is a change of behavior. Did you intend to do this in another PR?

Reply:
I don't see any change in behavior, am I missing smth?

Reply:
You are right -- there's no behavior change. There's a subtle difference that the new code won't check the type of the host op in PostOnStream, which I think is fine.

Reply:
IIUC the type of the host op is still checked, inside ´OptInDispatch::dispatch'